index.js

```diff
@@ -86,16 +86,20 @@ var deprecatedTargets = [
   {runtime: 'node', target: '1.1.0', abi: '43', lts: false},
   {runtime: 'node', target: '2.0.0', abi: '44', lts: false},
   {runtime: 'node', target: '3.0.0', abi: '45', lts: false},
-  {runtime: 'node', target: '8.0.0', abi: '52', lts: false},
   {runtime: 'electron', target: '0.30.0', abi: '44', lts: false},
   {runtime: 'electron', target: '0.31.0', abi: '45', lts: false},
   {runtime: 'electron', target: '0.33.0', abi: '46', lts: false}
 ]
 
-var allTargets = supportedTargets.concat(deprecatedTargets)
+var featureTargets = [
```

Comment:
English is not my native, but should not variable with items (Array) have ending `s`?